front/src/modules/apollo/hooks/useApolloFactory.ts

```diff
@@ -54,7 +55,7 @@ export function useApolloFactory() {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [setTokenPair, isDebugMode]);
 
-  useEffect(() => {
+  useUpdateEffect(() => {
```

Comment:
Not sure this is a good idea to introduce this hook. Because we are operating at a level where we want to force preventing rerendering, when we should look at the root cause of the rerenders.